get-started/searchgov-for-cloudgov-pages.md

```diff
@@ -0,0 +1,64 @@
+</ol>
+
+## On the Search.gov side of things:
+
+<ol><li>Sign up for a <a href="https://search.usa.gov/signup">user account</a>.</li>
+<li>Read our <a href="{{ site.baseurl }}/manual/site-launch-guide.html">Search Site Launch Guide</a> if you’d like some direction.</li>
+<li>Create and configure a search site in our <a href="https://search.usa.gov/sites">Admin Center</a>, as described in the <a href="{{ site.baseurl }}/manual/site-launch-guide.html">Search Site Launch Guide</a>. Highlights include:<br />
+<ul>
+<li><a href="{{ site.baseurl }}/manual/add-site.html">Add a search site</a></li>
+<li><a href="{{ site.baseurl }}/manual/brand.html">Brand your hosted search results page</a></li>
+</ul></li>
+<li>Request that we index your domain by <a href="mailto:search@gsa.gov">emailing our team</a>. Note, the site must be publicly available for our indexer to be able to access your content.</li>
+<li><a href="{{ site.baseurl }}/manual/preview.html">Preview your search results</a> once the indexing is complete.</li>
+</ol>
+
+## On the Cloud.gov Pages side of things, part 2
+
+<ol><li>Open the <code>_config.yml</code> file in your Cloud.gov Pages repo.</li>
+<li>Add your site's domain to the <code>url</code> field, approximately line 25. Leave this line commented out for now.<br />
+<ul>
+<li>If you're testing pre-production, enter your **demo** domain in the <code>url</code> field, and uncomment the line to allow the XML sitemap to build using this demo domain.</li>
+</ul>
+</li>
+<li>Find the <code>Search.gov configuration</code> section following the navigation and sidebar configuration sections. Add your site handle from the Admin Center to the <code>affiliate</code> line.</li>
+<li>If you are using the hosted search results page:<br />
+<ul>
+<li>Delete the value in the <code>access_key</code> line and/or comment out the line.</li>
+<li>Set the <code>inline</code> value to <code>false</code>.</li>
+</ul>
+</li>
+<li>If you're pulling search results into your website for display:<br />
+<ul>
+<li>Go to the Search.gov Admin Center, and find the <code>Activate</code> section, <code>API Access Key</code>. Copy this value to the <code>access_key</code>line in your Cloud.gov Pages <code>_config.yml</code>file.</li>
+</ul>
+</li>
+<li>If you do not want [type-ahead search suggestions]({{ site.baseurl }}/manual/typeahead-api.html) to appear in your website’s search box, change the <code>suggestions</code>value to <code>false</code>.</li>
+<li>When you're ready to launch your website, you will uncomment  the <code>url</code>line, to allow your XML sitemap to build using your production domain. Then, let our team know you're live and we'll begin indexing.<br />
+<ul>
+<li>If you tested using a demo domain, you'll need to update the <code>url</code>value to be your production domain.</li>
```

Comment:
Need spaces after the </code> tag
<img width="708" alt="image" src="https://user-images.githubusercontent.com/7243053/201171406-a83ba786-4568-4b3e-a3dc-0f16e2cb681e.png">